src/amo/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+
+export class InternalLinkBase extends React.Component {
+  static propTypes = {
+    base: PropTypes.string,
+    children: PropTypes.node,
+    to: PropTypes.string.isRequired,
+  }
+
+  render() {
+    const { base, children, to } = this.props;
+
+    return (<Link {...this.props} to={`${base}${to}`}>{children}</Link>);
```

Comment:
[path.join()](https://nodejs.org/api/path.html#path_path_join_paths) or `path.normalize()` might be handy to cope with double slashes when joining `base` and `to`.